vitest.config.ts

```diff
@@ -0,0 +1,22 @@
+import { defineConfig } from 'vitest/config';
+
+/**
+ * We are updating the coverage configuration to exclude some paths from the
+ * coverage report, so the coverage report only consider out code.
+ *
+ * We tried to define the vitest.config.ts file within the functions-subscription
+ * application, but it didn't work as expected.
```

Comment:
**suggestion (non-blocking)**: Consider adding more details. What was the problem? What didn't work as expected?
> but it didn't work as expected.